src/zenml/config/pipeline_spec.py

```diff
@@ -36,6 +36,10 @@ class PipelineSpec(FrozenBaseModel):
     version: str = "0.4"
     source: Optional[SourceWithValidator] = None
     parameters: Dict[str, Any] = {}
+    parameters_schema: Optional[Dict[str, Any]] = None
```

Comment:
Maybe these should be `input_schema` and `output_schema` or something along those lines? A pipeline does not have a `response`